WcaOnRails/app/views/shared/_associated_events_picker.html.erb

```diff
@@ -23,7 +23,7 @@ Note: form_builder.object is the object having associated events.
 
 <div class="form-group <%= "has-error" unless errors.empty? %>">
   <%= label_tag events_association_name, class: "associated-events-label" do %>
-    <%= label %>
+    <%= label %> (<span class="events-selected-count"></span> selected)
```

Comment:
This should be translated, which is quite problematic when we think of dynamic pluralization... Maybe there's some wording that doesn't depend on the count or at least would sound less weird. E.g. `selected count:`.